util/Migrator/MigratorConstants.cs

```diff
@@ -0,0 +1,7 @@
+namespace Bit.Migrator;
+
+public static class MigratorConstants
+{
+    public const string SqlTableJournalName = "Migration";
+    public const string DefaultMigrationsFolderName = "DbScripts";
+}
```

Comment:
⛏️ Newline. I know I am annoying with this but if you use something like VS Code you can install Prettier and it will be automatic if you format on save (recommended).